src/uct/ib/base/ib_md.h

```diff
@@ -157,6 +157,11 @@ typedef struct uct_ib_md {
         uint32_t             base;
         uint32_t             size;
     } mkey_by_name_reserve;
+    struct {
+        uint8_t              ooo_rw_rc;
+        uint8_t              ooo_rw_dc;
+        uint8_t              force;
```

Comment:
1. maybe these can be added as flags?
2. seems it should be in mlx5 devx md?

Reply:
fixed